model_compression_toolkit/gptq/pytorch/quantization_facade.py

```diff
@@ -209,6 +213,11 @@ def pytorch_gradient_post_training_quantization(model: Module,
 
         fw_impl = GPTQPytorchImplemantation()
 
+        # Attach tpc model to framework
+        attach2pytorch = AttachTpcToPytorch()
```

Comment:
Why not part of quantization core ?

Reply:
I don't want to expose it to the user as part of the API.
Do you think that we should?